app/controllers/api/v1/planning_applications_controller.rb

```diff
@@ -16,7 +16,7 @@ def index
 
       def show
         @planning_application = current_local_authority.planning_applications.where(id: params[:id]).first
-        if @planning_application
+        if @planning_application&.validated?
```

Comment:
Same as above - if you use a finder method that raises you can avoid all the logic